lms/templates/dashboard.html

```diff
@@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
       </ul>
     % else:
       <section class="empty-dashboard-message">
-        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
+        <p>${_("It looks like you aren't yet enrolled in any courses!")}</p>
```

Comment:
Something like this?

EdX offers hundreds of courses on subjects from computer science and personal finance to advanced Italian and data analysis. Learn from the world’s best institutions today!